documentation/docs/pages/operators/tokenomics/mixnet-rewards.mdx

```diff
@@ -44,33 +49,46 @@ To make it easier for the reader, we use a highlighting line on the left side, w
 </Callout>
 
 
-## Overview
+## Rewards Logic & Calculation
 
-This is a quick summary, to understand the full picture, please see detailed [*Rewards Logic & Calculation*](#rewards-logic--calculation) chapter below.
+This is a quick overview, to understand the logic behind fundamentals like [rewarded set selection](#rewarded-set-selection), [node performance](#node-performance-calculation), [stake saturation](#stake-saturation), or [rewards calculation](#rewards-calculation), please read the chapters below.
 
-* The operators of `nym-node` get rewarded from [Mixmining pool](https://validator.nymtech.net/api/v1/epoch/reward_params), which emits <span style={{display: 'inline-block'}}><EpochRewardBudget /></span> NYM per hour.
-* The [rewarded set](../tokenomics.mdx#active-set) of `nym-nodes` selected for Nym network routing and mixing is currently 240 nodes in total and it's selected for each new epoch (60 min). The number can be adjusted - look here for the current value: [validator.nymtech.net/api/v1/epoch/reward_params](https://validator.nymtech.net/api/v1/epoch/reward_params)
-* `nym-nodes` can run in mode `entry-gateway`, `exit-gateway` and `mixnode`, which are positioned into layers
-* NymVPN users can chose to route through Nym Network in two ways:
-	- Mixnet: 5 layers routing and mixing - full privacy
-	- Wireguard: 2 layers routing, skipping 3 mixing layers - fast mode
-* **The current reward system is [*Naive rewarding*](#naive-rewarding) - an intermediate step -  where each layer get's rewarded the same**
-* In the final model, nodes will get rewarded based on their layer position and the work they do (collected user tickets), where and the reward distribution per layer will be according to a [decision made by the operators](https://forum.nymtech.net/t/poll-what-should-be-the-split-of-mixmining-rewards-among-the-layers-of-the-nym-mixnet/407) as follows:
-    - 5-hop: 16%-16%-16%-16%-36%
-    - 2-hop: 33%-67%
-* Currently Gateways earn rewards only from taking a part in the rewarded set. The operators can sign up to a grant program as a substitution for 2-hop (Wireguad) routing.
-* To read more about the final design and future implementation, see [*Roadmap*](#roadmap) chapter for more details.
+* **The current reward system is called [*Naive rewarding*](#naive-rewarding) - an intermediate step - where the operators of `nym-node` get rewarded from [Mixmining pool](https://validator.nymtech.net/api/v1/epoch/reward_params), which emits <span style={{display: 'inline-block'}}><EpochRewardBudget /></span> NYM per hour**
+* **Only nodes selected to [rewarded set](../tokenomics.mdx#active-set) of Mixnet receive rewards**
+* The [rewarded set](../tokenomics.mdx#active-set) of `nym-nodes` is currently **240 nodes in total and it's selected for each new epoch (60 min)** 
```

Comment:
Suppose you might want to add from where / what the 240 are selected (and conversely what happens with nodes that aren't selected) ?